commitlint.config.ts

```diff
@@ -76,6 +76,14 @@ function isFooterNote(line: string): boolean {
         isFixesSentence(line);
 }
 
+function includesHashtagRef(text: string) {
+    return text.match(`#[0-9]+`) !== null;
+}
+
+function removeAllCodeBlocks(text: string) {
```

Comment:
@realmarv I thought that we had agreed already that this logic is already being used in other rules and then we should avoid code duplication?

Reply:
Sorry. Maybe it was in another branch and I didn't notice this one. Done